src/backend/joanie/core/serializers/admin.py

```diff
@@ -636,6 +636,30 @@ class Meta:
         read_only_fields = ["id", "course", "course_runs", "is_graded", "position"]
 
 
+class AdminOrderGroupSerializer(serializers.ModelSerializer):
+    """
+    Admin Serializer for OrderGroup model
+    """
+
+    nb_availabilities = serializers.SerializerMethodField(read_only=True)
```

Comment:
`available_seats` looks more easy to understand ? Without reading the method description I didn't understood what was the purpose of this parameter.

Reply:
This name is similar to the one used in the other OrderGroup serializer, to stay consistent

Reply:
when I see `available_seats` I understand that this is a list of the open seats.

Reply:
> when I see `available_seats` I understand that this is a list of the open seats.

And this is not the case ? In that case the docstring of the `get_nb_availabilities` is not accurate because it is exactly what it is written : `number of available seats for this order group`

Reply:
Through this discussion, this property has been renamed `nb_available_seats`